instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/grpcext/_interceptor.py

```diff
@@ -41,41 +39,67 @@ class _StreamClientInfo(
 ):
     pass
 
+def _get_metadata_timeout(**kwargs):
```

Comment:
This is being called as `_get_metadata_timeout(kwargs)` which would raise `_get_metadata_timeout() takes 0 positional arguments but 1 was given`.
```suggestion
def _get_metadata_timeout(kwargs):
```

Reply:
Unless it is fine to return `None` as the value for `metadata` and/or `timeout`, then this should not use `**kwargs`. This looks like we should be using positional arguments as before instead of using `**kwargs`.